python/src/deltachat/account.py

```diff
@@ -28,7 +29,7 @@ class Account(object):
     """
     MissingCredentials = MissingCredentials
 
-    def __init__(self, db_path, os_name=None, logging=True):
+    def __init__(self, db_path, os_name=None, logging=True) -> None:
```

Comment:
```suggestion
    def __init__(self, db_path: str, os_name: str = None, logging: bool = True) -> None:
```

Reply:
I am not sure about this, maybe this needs `os.PathLike`. And where it is `None` by default it is obviously not `str`, should be at least `Optional[str]`.

There are many more left to annotate, I have just annotated some as a proof that it works.

Reply:
> And where it is `None` by default it is obviously not `str`, should be at least `Optional[str]`

I think that is implied, if you say `a: str = None` that is the same as "optional str", at least I use it that way and mypy never complained